docs/content/setup/getting-started.md

```diff
@@ -6,5 +6,28 @@ To set up your instance follow these steps:
 1. Choose an installation method and follow the instructions
 2. [Configure your reverse proxy](https://docs.hedgedoc.org/guides/reverse-proxy/)
 3. [Configure HedgeDoc](https://docs.hedgedoc.org/configuration/)
+4. If you didn't disable [local accounts](/configuration/#email-local-account), you can use the "Sign In" button to
+   create an account, login and start using HedgeDoc.
 
 Follow us on <a href="https://social.hedgedoc.org/" target="_blank" rel="noreferer noopener">:fontawesome-brands-mastodon:{: .mastodon }Mastodon</a> or <a href="https://social.hedgedoc.org/twitter" target="_blank" rel="noreferer noopener">:fontawesome-brands-twitter:{: .twitter }Twitter</a> for updates.
+
+## Upgrading HedgeDoc
+
+HedgeDoc follows [Semantic Versioning](https://semver.org/).  
+This means that minor and patch releases should not introduce user-facing backwards-incompatible changes.
+
+You can find more details about upgrading in the instructions of your installation method.
+
+!!! warning
+    Before you upgrade, **always read the release notes**.  
+    You can find them on our [releases page](https://hedgedoc.org/releases/).
+
+## Migrating from CodiMD & HackMD
+Migrating from CodiMD <= 1.6.0 or HackMD <= 1.1.0 to HedgeDoc should be safe,
+just make sure to read the release notes.  
+A particular issue that has come up is when handling TLS connections when using a reverse proxy.
+You must [set the `X-Forwarded-Proto` header correctly](https://docs.hedgedoc.org/guides/reverse-proxy/).
```

Comment:
I think, it's best if the user updated to CodiMD 1.6.0 before making the switch… And do you mean HackMDs CodiMD or an super early version of HackMDs CodiMD still named HackMD?

Reply:
`CodiMD <= 1.6.0 or HackMD <= 1.1.0` is "our" code